src/blocks/weather/nws.rs

```diff
@@ -177,8 +185,13 @@ impl ApiForecast {
             (self.temperature.value - 32.0) * 5.0 / 9.0
         };
         let humidity = self.relative_humidity.value;
-        let wind_speed = self.wind_kmh();
-        australian_apparent_temp(temp, humidity, wind_speed)
+        let wind_speed = self.wind_kmh() / 3.6;
```

Comment:
Isn't this just `= self.wind_speed();`?

Reply:
Wind speed could be mph or m/s

Reply:
But wind_speed() is the function which you've added which always returns m/s?